control/xferfcn.py

```diff
@@ -10,6 +10,7 @@
 # Python 3 compatibility (needs to go here)
 from __future__ import print_function
 from __future__ import division
+from __future__ import absolute_import
```

Comment:
True, I just used my boilerplate "I need to make something Python 2/3 compatible" code. It can probably be removed.